.circleci/config.yml

```diff
@@ -93,6 +93,18 @@ jobs:
           name: Setup environment variables
           command: /build-tools-ci/scripts/set-environment
 
+      # Create theme npmrc.
+      - run:
+          name: Create .npmrc.
+          command: mv ./.ci/.npmrc-example ./.npmrc
+
+      # Authencate npmrc.
+      - run:
```

Comment:
I don't think this needs to be done. I believe we are only using public packages so so there is no reason to authenticate. We will need an .npmrc file so that npm knows to look at github, but I think the whole authentication part is not actually adding value for this project.